src/structure/Header.tsx

```diff
@@ -69,6 +70,17 @@ const EXTRA_HEADER_TAGS = {
       fg: themeDark.headerTagFg,
     },
   },
+  [APP_PATHS.AddNewChain]: {
+    title: 'Dashboard',
+    light: {
+      bg: GLOBALS.COLORS.GRADIENT_PRIMARY,
+      fg: themeLight.headerTagFg,
+    },
+    dark: {
+      bg: themeDark.headerTagBg,
+      fg: themeDark.headerTagFg,
+    },
+  },
```

Comment:
Confirm this from Zee the header text should also change for the new alias page.